tests/test_notifications.py

```diff
@@ -262,7 +261,7 @@ def test_node_subscriptions_and_backrefs_removed_when_node_is_deleted(self):
             subscription.save()
 
             s = getattr(project.creator, 'email_transactional', [])
-            assert_equal(len(s), 1)
+            assert_equal(len(s), 2)
```

Comment:
So that's how we encode subscriptions. How do we process subscriptions when sending mail?